CMakeLists.txt

```
@@ -616,6 +616,14 @@ if(BUILD_TEST)
   add_test_without_main(test_multidevice "${MULTIDEVICE_TEST_SRCS}" "")
   list(APPEND TEST_BINARIES test_multidevice)
 
+  set(MULTIDEVICE_TUTORIAL_SRCS)
+  list(APPEND MULTIDEVICE_TUTORIAL_SRCS
+    ${NVFUSER_ROOT}/tests/cpp/multidevice.cpp
+    ${NVFUSER_ROOT}/tests/cpp/test_multidevice_tutorial.cpp
```

```suggestion
    ${NVFUSER_ROOT}/tests/cpp/test_multidevice_executor_tutorial.cpp
```
Nit: at some point, I'll try to add a tutorial for FusionExecutorCache with multidevice, so I slightly prefer naming it the MultiDeviceExecutor tutorial. But we could always rename things on demand.

---

I prefer calling it "test_multidevice_tutorial". I am not focussing on the multidevice executor in the test.

I suggest when FusionExecutorCache is ready we will use it those tests